tutorons/python/detect.py

```diff
@@ -0,0 +1,62 @@
+#! /usr/bin/env python
+# -*- coding: utf-8 -*-
+
+from __future__ import unicode_literals
+import logging
+import ast
+
+from tutorons.common.extractor import Region
+from tutorons.python.explain import explanations
+
+logging.basicConfig(level=logging.INFO, format="%(message)s")
+
+
+def filter_non_ascii(c):
+    if ord(c) > 127:
+        return ' '
+    return c
+
+
+def findOffset(text_list, call):
+    """Takes in a text as a list and function call and finds the start of call within text"""
+    offset = 0
+    if call.lineno > 1:
+        for l in xrange(call.lineno - 1):
```

Comment:
Why use `xrange` instead of `range`, which performs the same thing but has a simpler signature?